content-scripts/src/modules/options/all.js

```diff
@@ -0,0 +1,130 @@
+import {
+  changeFollowCount,
+  changeLatestTweets,
+  changeLikeCount,
+  changePromotedPosts,
+  changeReplyCount,
+  changeRetweetCount,
+  changeSearchBar1,
+  changeSearchBar2,
+  changeTopicsToFollow,
+  changeTweetButton,
+  changeWhoToFollow,
+} from "./interface";
+import {
+  changeBookmarksButton,
+  changeCommunitiesButton,
+  changeExploreButton,
+  changeHomeButton,
+  changeListsButton,
+  changeMessagesButton,
+  changeNavigationButtonsLabels,
+  changeNavigationButtonsLabelsHover,
+  changeNavigationCenter,
+  changeNotificationsButton,
+  changeProfileButton,
+  changeTopArticlesButton,
+} from "./navigation";
+import {
+  changeTimelineBorders,
+  changeTimelineWidth,
+  changeTweetBorders,
+} from "./timeline";
+import { changeGrowButton, changeTypefullyComposerButtons } from "./typefully";
+import { changeWriterMode } from "./writer-mode";
+
+// Array of user preferences, passed to `injectAllChanges`
+export const userPreferences = [
+  "timelineWidth", // 1
```

Comment:
@thomaswang  Can you remove all these `//` comments? If these don't need to be ordered, we don't need them anyway